platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r32/BuildFinishedCrossVersionSpec.groovy

```diff
@@ -19,12 +19,17 @@ package org.gradle.integtests.tooling.r32
 import org.gradle.integtests.tooling.fixture.TargetGradleVersion
 import org.gradle.integtests.tooling.fixture.ToolingApiSpecification
 import org.gradle.tooling.model.GradleProject
+import org.gradle.util.GradleVersion
 
 @TargetGradleVersion(">=3.2")
 class BuildFinishedCrossVersionSpec extends ToolingApiSpecification {
 
     def "model builders are executed before buildFinished hook"() {
+        println("releasedGradleVersion: " + targetVersion)
```

Comment:
🤔 Is this change intentional or a leftover?

Reply:
oops, good catch